dWeb/Web.cpp

```diff
@@ -0,0 +1,292 @@
+#include "Web.h"
+#include "Game.h"
+#include "magic_enum.hpp"
+#include "json.hpp"
+#include "Logger.h"
+#include "eHTTPMethod.h"
+#include "GeneralUtils.h"
+#include "JSONUtils.h"
+
+namespace Game {
+	Web web;
+}
+
+namespace {
+	const char* json_content_type = "Content-Type: application/json\r\n";
```

Comment:
There seems to be a mix of snake_case and camelCase in the file, could you switch this and any other variables over to camelCase as we do everywhere else?